src/sentry/features/manager.py

```diff
@@ -49,6 +51,15 @@ def get(self, name, *args, **kwargs):
         cls = self._get_feature_class(name)
         return cls(name, *args, **kwargs)
 
+    def add_default_handler(self, handler, backup_handler=None):
```

Comment:
Would a name like 'entity type' handler be applicable here? Having default and backup in the same class is a bit confusing to me.